XdfStreamer/xdfstreamer.cpp

```diff
@@ -62,7 +60,7 @@ void XdfStreamer::pushRandomData(QSharedPointer<lsl::stream_outlet> outlet_ptr,
             }
         }
 
-        while (((double)clock()) / CLOCKS_PER_SEC < starttime + t * dSamplingInterval);
+        std::this_thread::sleep_for(std::chrono::milliseconds(dSamplingInterval));
```

Comment:
Shouldn't this be removed now that line 104 is in?

Reply:
oh sorry, I was looking at a partial diff. This is fine.

Reply:
This one will still have the problem discussed previously (eventually fewer samples pushed than expected based on elapsed time).

Something like:
`std::this_thread::sleep_for(std::chrono::millseconds(1000* (starttime + t*dSamplingInterval - ((double)clock()/CLOCKS_PER_SEC))));`

Please check.